python/tests/test_ibis.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import textwrap
+
+import cudf
+import cupy
+import ibis
+
+from legate_dataframe import LogicalTable
+from legate_dataframe.experimental_ibis import LegateBackend
+from legate_dataframe.testing import assert_frame_equal
+
+
+def test_ibis_basic_read_csv_and_binary(tmp_path):
+    file = tmp_path / "test.csv"
+    file.write_text(
+        textwrap.dedent(
+            """\
+        a,b,c
+        1,2,3
+        2,3,4
+        """
+        )
+    )
+
+    schema = ibis.schema({"a": "int64", "b": "float64", "c": "int64"})
+    b = LegateBackend().connect()
```

Comment:
```suggestion
    b = LegateBackend.connect()
```

The `connect` call will return a new class object anyway, so no need to instantiate the backend class separately.

Reply:
Hmmm, realized I hadn't applied this, and just tried it.  But it seems `connect()` is a normal method and the instantiation is necessary?